syntax				= "proto3";
option optimize_for = CODE_SIZE;

import "common/AttribChangeDataProto.proto";

message Cfg_Status
{

		uint32						   id				  = 1;	//
		uint32						   level			  = 2;	//状态等级
		uint32						   status_type		  = 3;	//状态类型编号
		uint32						   status_expire_type = 4;	//状态过期类型
		uint32						   status_flag		  = 5;	//状态掩码
		uint32						   power			  = 6;	//数值
		int32						   secs				  = 7;	//每隔N秒执行1次
		int32						   times			  = 8;	//总共执行N次
		uint32						   max_times		  = 9;	//最大可叠加次数
		uint32						   max_secs			  = 10; //最大可叠加秒数
		int64						   scriptid			  = 11; //脚本编号
		repeated AttribChangeDataProto attrib_change_list = 12;

}